New version: NNlib v0.9.29

JuliaRegistrator opened the pull request:
- Registering package: NNlib
- Repository: https://github.com/FluxML/NNlib.jl
- Created by: @ToucheSir
- Version: v0.9.29
- Commit: 85d80d1c4cf87589a52a755dc155b0c816537031
- Reviewed by: @ToucheSir
- Reference: https://github.com/FluxML/NNlib.jl/commit/85d80d1c4cf87589a52a755dc155b0c816537031#commitcomment-153946910
- Description: Neural Network primitives with multiple backends
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6e892c0f7c7353f7cda31826e5e7eb452a1e34a1c63ae2e8efa083694d0ec27a4bd8fe3ca3a1eeaba021e54e9397b46eca19da2b23f5e88fbc1124b5e0c8444ce18356ce21b23d53c9ef86bf4b3dc7bc4b2c84f5a07de8caf6a5a32f5c33cd191ad08ee3688b8b8a8abdf81e4f7be1f7e1f99f7867c062410d7721a2748a114be723d2185204df3275366ad63f477c3f1ba7c313f7890be266aea2c9aff634d3ae0883f77ed592f3a9cb9d2aedd0d53c2 -->

ToucheSir commented:
Note: this was made in error (we accidentally skipped registering a version), and no further action is required before the auto-close happens.

JuliaRegistrator committed:
New version: NNlib v0.9.29

UUID: 872c559c-99b0-510c-b3b7-b6c96a88d5cd
Repo: https://github.com/FluxML/NNlib.jl.git
Tree: e8d4268b4a438bdad98937e4bca45881363b4767

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
